src/main/java/blackjack/controller/BlackjackController.java

```diff
@@ -0,0 +1,126 @@
+package blackjack.controller;
+
+import blackjack.domain.*;
+import blackjack.view.InputView;
+import blackjack.view.OutputView;
+
+public class BlackjackController {
+
+    public void run() {
+        OutputView.printPlayerNameInstruction();
+
+        CardMachine cardMachine = new CardMachine();
+        Dealer dealer = new Dealer();
+        Players players = createPlayers();
+
+        noticeInitCard(cardMachine, dealer, players);
+        openInitCard(dealer, players);
+
+        distributeCardToPlayers(players, cardMachine);
+        distributeCardToDealer(dealer, cardMachine);
+
+        openResult(dealer, players);
+        win(dealer, players);
+    }
+
+    private Players createPlayers() {
+        try {
+            return new Players(InputView.inputPlayerName());
+        } catch (IllegalArgumentException exception) {
+            OutputView.printExceptionMessage(exception.getMessage());
+            return createPlayers();
+        }
+    }
+
+    private void noticeInitCard(CardMachine cardMachine, Dealer dealer, Players players) {
```

Comment:
메서드가 가지는 행위에 비해 메서드 명이 모호한 것 같아요. 출력문 상관없이 해당 메서드의 가장 핵심적인 행위는 처음 카드 2장을 나눠주는 것이라고 보입니다. 이에 맞게 적절히 메서드 명을 작성해보면 좋을 것 같아요!! :)

Reply:
```suggestion
    private void dealInitCards(CardMachine cardMachine, Dealer dealer, Players players) {
```
이름을 dealInitCards로 바꿨어요.